contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -184,6 +215,37 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     return s_nodeOperators[nodeOperatorId];
   }
 
+  /// @notice Adds nodes
+  /// @param nodes The nodes to add
+  function addNodes(Node[] calldata nodes) external {
```

Comment:
Are we going to support updating capabilities of a node? Or will that require removal+re-addition? That might be a bit disruptive.

Reply:
We'll have support for updates - will come in a follow-up PR.